metaseq/scripts/download_opt175b.sh

```diff
@@ -17,7 +17,7 @@ do
   for shard_id in $(seq 0 123)
   do
     filename="checkpoint_last-model_part-$part_id-shard$shard_id.pt"
-    wget -O "$filename" "${presigned_url/$str_to_replace/$filename}"
+    wget -O "$2$filename" "${presigned_url/$str_to_replace/$filename}"
```

Comment:
Nit: add a comment above indicating the use of this second arg?

Reply:
Ooops! I did add that comment to my repo but it seems to have happened in another branch. I’ll chose this PR, merge on my end, and reopen it